tuxemon/resources/db/locale/eo.json

```diff
@@ -5,18 +5,18 @@
     "combat_used_x_on_y": "${{name}} estis uzata al ${{target}} de ${{user}}!",
     "combat_call_tuxemon": "Iru, ${{name}}!",
     "combat_opponent_call_tuxemon": "${{user}} sendas ${{name}}!",
-    "combat_state_poison_damage": "${{name}} estis venenigita!",
-    "combat_status_damage": "${{name}} estis damaĝita de ${{status}}!",
+    "combat_state_poison_damage": "${{name}} veneniĝis!",
+    "combat_status_damage": "${{name}} damaĝiĝis pro ${{status}}!",
     "combat_victory": "Vi gajnis!",
```

Comment:
'Gajni' usually means 'acquire' or 'win' (in the sense of winning a prize). E.g. "Mi gajnis la grandan gajnon".
Although it can be used in the sense of "I won a game" (Mi gajnis ludon), I think it would be nicer to use 'venki' here because conveys the meaning of overcoming the opponent and it has nicer symmetry with the following string, which uses 'venki'.
Suggestion: "Vi venkis!"